setup.py

```diff
@@ -106,7 +106,7 @@ def run(self):
               'h5io'
           ],
           extras_require={
-              'gui': ['ipywidgets <=7.7.1', 'ipympl<0.9', 'voila<=0.3.6'],
+              'gui': ['ipywidgets>=8.0.0', 'ipykernel', 'ipympl', 'voila'],
```

Comment:
I think you'll need to update README:

https://jonescompneurolab.github.io/hnn-core/stable/index.html

let me know once you fix it and I'll go ahead and merge @gtdang !

Reply:
Thanks! Updated!